### PR TITLE
Adds React CLI example integration

### PR DESCRIPTION
**Pull request by wallynm**
The idea to create create this example folder is to help guys that already understand React into how they can integrate with OrbitDB.

I've replicated the same logic located into the vanilla implementation, where the dev can download and install the packages and everything works out of the box.

To achieve that i've implemented two frameworks explained into Readme.md file
If there's something that i could improve, just send me
Would be awesome to have this kind example at the repo!

### CODE REVIEW

**Comment by phillmac** [comment]
@wallynm `ERROR in Entry module not found: Error: Can't resolve './examples/browser/browser-webpack-example/index.js' in '/home/circleci/orbitdb/orbit-db'`

You might need to fix the webpack config to handle the moved examples dir structure

**Comment by wallynm** [comment]
Im currently working to fix this!
I would like to propose to have all examples centralized into the examples folder, today we have other packages that i didn't even know, just get to know then because another issue that i was googling thats electron app:
https://github.com/orbitdb/orbit-electron/issues

If we had everything inside the examples folder, as this one im updating
We could have other guys implementing several config examples

**Comment by wallynm** [comment]
I'm still finishing this PR so it don't break the base tests.
I think i will have time to finish this into this weekend, ty guys (:

**Comment by aphelionz** [comment]
I'm not sure the tests are your fault. Running them locally now to check.

**Comment by aphelionz** [comment]
Ha, nope. I was wrong. You likely know what you need to do but just in case - you'll need to build orbitdb to be included in each of the example folders, OR copy the dist files to each of them. That might be cleaner?

**Comment by aphelionz** [comment]
Hey @wallynm thanks again for your effort here. I'll be looking to get the 0.24.0 release out sooner rather than later so let me know what I can do to help here

**Comment by aphelionz** [comment]
Hey @wallynm checking in here - we just released 0.25.0. Want to take another stab at this?

**Comment by wallynm** [comment]
Hi @aphelionz, i've been following the recent updates! I will take another shot this week, sorry for taking so long but i'm better to handle these errors this time! I let you know!

**Comment by aphelionz** [comment]
Thanks @wallynm

**Comment by wallynm** [comment]
Ok so, @aphelionz i did have some improvement here ahaha
First, im so glad that you've waited for so long and didn't closed the PR as several other guys would do, thank you so much and i feeling so good to finally can finish this PR!

First, I would appreciate some help into those Readme files because english isn't my first language, and i think it might have several language mistakes.

Second, for the check mark, i've removed the build examples phase to see if it was the only problem running, and turns out that was the problem - basically because the examples structure had change, so i need to figure out how to fix this one and make sure the examples get testes as well, at least the build phase as we have support right now.

I've also updated for latest versions the examples, so we have they all working correctly as expected.
Lastly, thank you so much for waiting so long ahaha
Tomorrow i will try to understand how to finish this last part about examples folder not correctly building.

**Comment by aphelionz** [comment]
Happy to review the language in the READMEs, and thank YOU for jumping back in! :+1:

File: examples/vanilla-cli-latest-ifps/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Vanilla Cli Latest IFPS
+
+This project uses the latest version of IFPS, which have a different way to initiate the IFPS instance, you can compare both versions([IFPS old version](https://github.com/orbitdb/orbit-db/blob/master/examples/vanilla-cli)) and see how to configure each one and compare their differences.
```
Comment (aphelionz) [inline examples/vanilla-cli-latest-ifps/README.md:3]:
```suggestion
This project uses the latest version of IFPS, which has a different way to initiate the IFPS instance. You can compare both versions([IFPS old version](https://github.com/orbitdb/orbit-db/blob/master/examples/vanilla-cli)) and see how to configure each one and compare their differences.
```

File: examples/vanilla-cli-latest-ifps/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Installation
+```bash
+$ yarn install
+# or
+$ npm install
+```
+
+## Usage
+The project already configured with a basic webpack bundler config, generates a build folder where you can test any of the following commands:
```
Comment (aphelionz) [inline examples/vanilla-cli-latest-ifps/README.md:14]:
```suggestion
The project is already configured with a basic webpack bundler config. It generates a build folder where you can test any of the following commands:
```

File: examples/vanilla-cli/readme.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Vanilla Cli
+This project uses old version of IFPS (0.40), which have a different way to initiate the IFPS instance, you can compare both versions([IFPS latest version](https://github.com/orbitdb/orbit-db/blob/master/examples/vanilla-cli-ifps-latest)) and see how to configure each one and their differences.
```
Comment (aphelionz) [inline examples/vanilla-cli/readme.md:2]:
```suggestion
This project uses old version of IFPS (0.40), which has a different way to initiate the IFPS instance. You can compare both versions([IFPS latest version](https://github.com/orbitdb/orbit-db/blob/master/examples/vanilla-cli-ifps-latest)) and see how to configure each one and their differences.
```

File: examples/vanilla-cli/readme.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## Installation
+```bash
+$ yarn install
+# or
+$ npm install
+```
+
+## Usage
+The project already configured with a basic webpack bundler config, generates a build folder where you can test any of the following commands:
```
Comment (aphelionz) [inline examples/vanilla-cli/readme.md:12]:
```suggestion
The project is already configured with a basic webpack bundler config. It generates a build folder where you can test any of the following commands:
```

**Pull request closed without merging**